### ISSUE TITLE
Add support for `image`, `imageReference`

### ISSUE THREAD

**Issue by rjhilgefort** [remarkjs/remark-validate-links#29]
Doesn't look like it tries to validate images that are specified to be auto-expanded and rendered with the `![](image.jpg)` syntax. Given that `foo.jpg` doesn't exist:

This doesn't show a warning:
```
![](image.jpg)
```

And this does:
```
[](image.jpg)
```

**Comment by wooorm** [comment]
Hmm, correct, [only `link` and `linkReference` are supported](https://github.com/remarkjs/remark-validate-links/blob/7187b4b74ba3045fca58e66ff0fe446d55949451/index.js#L277).

Could you expand on why you expected this to be supported?

**Comment by rjhilgefort** [comment]
I guess I just assumed that if `[](image.jpg)` was checked than `![][image.jpg]` would be checked too. My apologies.

**Comment by wooorm** [comment]
Don’t worry, was just wondering what your reasoning was. I think it makes sense to add it!

**Comment by rjhilgefort** [comment]
❤️ Thanks for adding it!

**Comment by wooorm** [comment]
Sure, thanks for raising the issue 👍

I’m waiting ’till https://github.com/remarkjs/remark-validate-links/issues/26 is resolved (which has an external dependency), and would also like to see what happens in https://github.com/remarkjs/remark-validate-links/issues/28, before I merge it though.

If I forget and it’s important to you feel free to ping again in a week or so and I’ll move things along anyway!

**Comment by rjhilgefort** [comment]
Sounds good!
